demo/src/app/components/typeahead/overview/sections/user-interaction/template.html

```diff
@@ -0,0 +1,82 @@
+<h5>Edit the search term</h5>
+
+<p>
+  The classic usage of the Typeahead is to use the input to edit the search term, and trigger a search process which will spawn a new options list.
+</p>
+
+<p>
+  Apart from creating an Observable based on the <code>input</code> event of the input, the Typeahead does nothing else specific, so to edit the search term just use it as any other input.
+</p>
+
+
+
+<h5>Navigate through options in the popup</h5>
```

Comment:
Ha so here we go... I struggled finding proper naming, and I am still not satisfied.

Initially, I wanted to say _select_, but this is what we use to say that we choose the option / commit it to the model.

Then _highlight_, but I found it confusing with the `ngb-highlight` component, whose role is to highlight part of the option. Also, "highlight" is a word related to visual appearance only, so I find it's not obvious it is related to where the "cursor" (by cursor I mean a current position in the list). And if we ever had a feature to highlight the currently selected/committed option, this would even be more confusing, so I don't like this term.

So to sum up: 

- _activate_: not very clear, we both agree
- _select_: good but already taken to say we choose an option
- _highlight_: in the current case works because indeed putting the (virtual) cursor on an option highlights it, but to me it's like using a side effect to represent the thing (e.g. if we said "to outline" instead of "to focus", while the outline is a result of a focus)

Still thinking...